src/driver.rs

```diff
@@ -89,7 +120,7 @@ impl Driver {
                 size_y as c_int,
                 bands as c_int,
                 T::gdal_type(),
-                null_mut(),
+                options_c as *mut *mut i8,
```

Comment:
shouldn't this be a *mut * mut char?

Reply:
You mean `c_char`?

Reply:
yes, `options_c as *mut *mut i8` is correct. I'm not sure how it ended up being `null_mut()` again, sorry.

Reply:
It's not `null_mut()`, it's `options_c as *mut *mut i8`, jdroenner is asking if it should be `options_c as *mut *mut c_char`.

Reply:
oh, sorry! my bad.  Blame Friday night.

Reply:
gdal_sys::GDALCreate expects *mut *mut i8.

Reply:
Yeah, it's defined as `char**` in the GDAL headers. We might be able to improve the bindings, but I think we shouldn't block merging this.

Reply:
Any thoughts @jdroenner?  Does the bindings need to be updated before merging this? I think I don't have enough knowledge to do that.

Reply:
no its fine. i just assumed it to be a char in the bindings.

Reply:
Cool! Thanks!